<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="end">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Avsluta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="handelOrders">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hantera Ordrar</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="showAllProducts">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addOrder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ändra saldo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeProduct">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ta Bort Produkt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addProduct">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lägg till Produkt</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="showStorage">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarstorag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hur fullt är lagret</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NrOfOrders">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>179</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ShowAll">
       <property name="text">
        <string>Visa alla produkter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clear">
       <property name="text">
        <string>Rensa</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>158</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="showStorageButton">
       <property name="text">
        <string>Visa Lagret</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearStorage">
       <property name="text">
        <string>Rensa</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
